packages/millicast-sdk/src/workers/TransformWorker.worker.js

```diff
@@ -59,9 +95,9 @@ addEventListener('rtctransform', (event) => {
     codec = event.transformer.options.codec
     transform = createSenderTransform()
   } else if (event.transformer.options.name === 'receiverTransform') {
-    codecMap = event.transformer.options.codecMap || {}
+    payloadTypeCodec = event.transformer.options.payloadTypeCodec || {}
```

Comment:
nitpick non-blocking : `payloadCodecType`

Reply:
The thing is that is the codec's payload type, so these are two different attributes